ts/features/payments/bizEventsTransaction/screens/PaymentsTransactionBizEventsDetailsScreen.tsx

```diff
@@ -121,6 +124,16 @@ const PaymentsTransactionBizEventsDetailsScreen = () => {
     );
   };
 
+  useHeaderSecondLevel({
+    title:
+      transactionDetails?.carts?.[0].payee?.name ||
```

Comment:
I prefer using nullish coalescing operator (`??`) as it is a safer operator instead of a logical or (`||`)

Reply:
Addressed into 74ba39d94277188ece31672b88fcc1fbca5e0ea6